DataFormats/METReco/src/classes_def.xml

```diff
@@ -164,11 +164,15 @@
   </class>
   <class name="edm::Wrapper<reco::EcalHaloData>"/>
```

Comment:
Hi Slava, I have added a line for HaloTowerStrip. It now passes `runTheMatrix.py -l 1102 -i all`:
https://github.com/cms-sw/cmssw/pull/11447

Similarly, I have updated 74X, 75X versions with the same exact changes